test/CMakeLists.txt

```diff
@@ -24,9 +22,45 @@ if(MANIFOLD_CBIND)
   list(APPEND SOURCE_FILES manifoldc_test.cpp)
 endif()
 
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+# Prevent installation of GTest with your project
+set(INSTALL_GTEST OFF CACHE BOOL "" FORCE)
+set(INSTALL_GMOCK OFF CACHE BOOL "" FORCE)
+
+find_package(GTest 1.12)
+if((NOT GTest_FOUND) OR NOT(GTest_VERSION GREATER_EQUAL 1.12))
+  message(STATUS "GTest not found, downloading from source")
```

Comment:
@kintel - what do you think of this approach? Would this make consuming our library easier for the likes of OpenSCAD?

Reply:
Looks reasonable. It does make the build system depend on having access to github.com, which is a known issue from some countries, as well as for people with limited connectivity, but that's a separate challenge. For OpenSCAD we'll likely try to hit the find_package call, so this should work well for us!

Reply:
Yes it depends on access to github by default, but if for some reason that is not possible and `find_package` still cannot find the package, you can specify the directory by setting `-DFETCHCONTENT_SOURCE_DIR_GOOGLETEST`. I tried using this for the nix package (it blocks internet connection when building) and it works.

Reply:
Awesom! Not familiar with FetchContent..

Reply:
Nice - would be good to add that trick to the README somewhere.